integration_tests/http/raw-path.yaml

```diff
@@ -0,0 +1,13 @@
+id: basic-raw-example
+
+info:
+  name: Test RAW Template
```

Comment:
The `id` and `name` should mention trailing forward slash. I'd suggest something like this:
```
id: raw-with-path-final-forward-slash
name: Test Raw Request with a final forward slash in the path
```